Makefile

```diff
@@ -35,5 +35,17 @@ refresh-postgres-data:
 	@docker-compose up -d postgres
 	@sleep 2
 
+build-services:
+	@docker-compose build --no-cache
+
 up:
-	docker-compose up
+	@docker-compose up -d
+
+down:
+	@docker-compose down --volumes
```

Comment:
volumes are usually persisted, are you sure you want to delete database data each time?

Reply:
`docker-compose down` also removes database data

Reply:
it shouldn't, down only stops and removes containers and networks by default
it won't do anything with volumes, that's why you're providing an extra flag here

Reply:
context: https://docs.docker.com/reference/cli/docker/compose/down/